src/styles/blocks/stars.css

```diff
@@ -0,0 +1,20 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(../../images/star.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+  width: 16px;
+  height: 16px;
+}
+
+.stars__star:last-child {
+  margin-right: 0;
+}
+
+.stars--4 .stars__star:nth-child(-n+4) {
+  background-image: url(../../images/star-active.svg);
```

Comment:
Check task with stars, if you change modifier, you need to receive same quantity of yellow stars ( for example: stars--2 = 2 stars)